src/modules/displaymanager/main.py

```diff
@@ -849,6 +850,8 @@ def set_autologin(self, username, do_autologin, default_desktop_environment):
         de_command = default_desktop_environment.executable
         if os.path.exists(self.os_path("usr/bin/gtkgreet")) and os.path.exists(self.os_path("usr/bin/cage")):
             self.config_data['default_session']['command'] = "cage -d -s -- gtkgreet"
+            if self.greeter_css_location:
+                self.config_data['default_session']['command'] += " -s {self.greeter_css_location}"
```

Comment:
Oops, there is a typo I here.  I missed the `f`.  It should be:
```
                self.config_data['default_session']['command'] += f" -s {self.greeter_css_location}"
```

Reply:
Done